packages/gigamon/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.7.0"
+  changes:
+    - description: Child Dashboards added for ZT and attribute change in a OT widget.
```

```suggestion
    - description: Child Dashboards added for ZT and attribute change in an OT dashboard widget.
```
But I think the "attribute change" should be explained. What was changed and why?

---

Suggest instead:
```
Added child dashboards for ZT and updated "OT Sessions with Latency" widget to use `gigamon.ami.dst_mac` instead of `gigamon_ami.dhcp_request.dst_mac`.
```

---

Or, what I've now added as a suggestion.